tsconfig.json

```diff
@@ -1,17 +1,50 @@
 {
     "compilerOptions": {
         "target": "es6",
-        "lib": ["es2015", "dom"],
+        "lib": ["dom"],
+        "skipLibCheck": false,
         "module": "commonjs",
         "outDir": "dist",
-        "alwaysStrict": true,
+        "alwaysStrict": false,
+        "strict": false,
+        "composite": true,
         "incremental": true,
         "sourceMap": true,
         "noImplicitAny": true,
+        "noImplicitReturns": false,
         "removeComments": true,
         "noLib": false,
         "preserveConstEnums": true,
         "experimentalDecorators": true,
-        "declaration": true
-    }
+        "declaration": true,
+        "esModuleInterop": true,
+        "strictFunctionTypes": false,
+        "moduleResolution": "node" /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */,
+        "resolveJsonModule": false,
+        "noStrictGenericChecks": false,
+        "noImplicitThis": true,
+        "skipDefaultLibCheck": false,
```

Comment:
I don't remember exactly where this came from but I think is something particular that should stay on the specific package.

Reply:
This flags uses the default value from typescript. we can remove them here, or keep them to make thing more explicit. 
Let me know what works best for you.

Reply:
I prefer having only configurations that differ from the default, what do you think @danielpeintner ?

Reply:
I agree with @relu91. I suggest we use flags only when our need does not not match the default.